.github/workflows/test.yml

```diff
@@ -236,8 +236,72 @@ jobs:
           E2E_CI_TIMEOUT_SEC: '600'
           E2E_KATHY_MESSAGES: '20'
 
-      - name: cli e2e tests
-        run: ./typescript/cli/ci-test.sh
+  cli-e2e:
+    runs-on: larger-runner
+    needs: [yarn-build]
+    strategy:
```

Comment:
This is a lot of extra CI code just to run test with two values for hook. Can you add this logic into the CI script instead of here? I've found CI configs are brittle and doesn't get maintained very well so I'd rather keep it simple

Reply:
for another issue for now